File: Windows/OpenIoT.App/OpenIoT.Lib/Web/Models/PresetsCollection.cs
namespace OpenIoT.Lib.Web.Models
{
    public class PresetsCollection : List<Preset>
    {
        public PresetsCollection()
        {

        }

        public PresetsCollection(IEnumerable<Preset> presets)
        {
            this.AddRange(presets);
        }
    }
}

File: Andriod/Constructor/construction/src/main/java/com/palitri/openiot/construction/framework/web/models/PresetsCollection.java
package com.palitri.openiot.construction.framework.web.models;

import java.util.ArrayList;

public class PresetsCollection extends ArrayList<Preset> {
    public Preset GetPresetById(String presetId)
    {
        for (Preset preset : this)
            if (preset.projectPresetId.equals(presetId))
                return preset;

        return null;
    }
}
